<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AirShower3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(128, 128, 128, 255));</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="GLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(50, 50, 50);</string>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_animation">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Animation: 1.0x</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSlider" name="sliderAnimationSpeed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkLoop">
            <property name="text">
             <string>Loop</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPauseButton">
            <property name="text">
             <string>Play/Pause</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Elapsed Time:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSlider" name="sliderTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_components">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Airshower Components:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_em">
            <property name="text">
             <string>el. magnetic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_mu">
            <property name="text">
             <string>muonic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_hd">
            <property name="text">
             <string>hadronic</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_detector">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(25, 25, 25);</string>
         </property>
         <property name="title">
          <string>Detektors:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_radius">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBox_radius">
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_levels">
            <property name="text">
             <string>Crowns:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_levels">
            <property name="maximum">
             <number>23</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_distance">
            <property name="text">
             <string>Distance:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBox_distance">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>3.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="onlyShowerFront">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Show only shower front</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lookatShowerFront">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Camera follows shower front</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>groupBox_time</zorder>
      <zorder>groupBox_animation</zorder>
      <zorder>groupBox_detector</zorder>
      <zorder>onlyShowerFront</zorder>
      <zorder>lookatShowerFront</zorder>
      <zorder>groupBox_components</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>gui/glwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
